Doc/library/sqlite3.rst

```diff
@@ -25,6 +25,15 @@ The sqlite3 module was written by Gerhard Häring.  It provides an SQL interface
 compliant with the DB-API 2.0 specification described by :pep:`249`, and
 requires SQLite 3.7.15 or newer.
 
+This document includes four main sections:
+
+* :ref:`sqlite3-tutorial`: teaches how to use the sqlite3 module
+* :ref:`sqlite3-reference`: describes the classes and methods defined in the
+  module
```

Comment:
```suggestion
* :ref:`sqlite3-reference` describes the classes and functions
  this module defines
```

* "methods" -> "functions" since "classes" include "methods" whereas much of the reference docs cover top-level functions
* Reduce wordy repetition
* Phrase as prose sentence